armory/instrument/config.py

```diff
@@ -0,0 +1,428 @@
+"""
+Set up the meters from a standard config file
+"""
+
+import numpy as np
+
+from armory.instrument.instrument import (
+    LogWriter,
+    Meter,
+    ResultsWriter,
+    get_hub,
+)
+from armory.logs import log
+from armory.utils import metrics
+
+
+class MetricsLogger:
+    """
+    Uses the set of task and perturbation metrics given to it.
+    """
+
+    def __init__(
+        self,
+        task=None,
+        task_kwargs=None,
+        perturbation=None,
+        means=True,
+        include_benign=True,
+        include_adversarial=True,
```

Comment:
need to dig further, but I'm noticing that `include_adversarial` is `True` even when I set `--skip-attack`

Reply:
`include_targeted` is also `True` even though I'm not doing a targeted attack. Again, not sure yet what implications this has if any

Reply:
I think those will still run fine and not output anything.

Reply:
are we expecting users to potentially set `include_benign` or `include_adversarial` to `False` in the config file (as this is the only way to override the default `True`)?

Reply:
I switched this back to the prior way, where the scenario flags would get passed in at construction.

Reply:
Just pushed an update.